src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/JsonSerializerContextTests.cs

```diff
@@ -380,6 +380,21 @@ public static void SupportsGenericParameterWithCustomConverterFactory()
             Assert.Equal(@"[""Cee""]", json);
         }
 
+        // Regression test for https://github.com/dotnet/runtime/issues/74652
```

Comment:
Could you also add a test validating https://github.com/dotnet/runtime/issues/61734?

Reply:
didn't know there was another issue